app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -360,6 +360,34 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
     border: 1px solid var(--base-10);
   }
 
+  .current-user-info {
+    text-align: center;
+    margin-bottom: 1rem;
+
+    .current-user-avatar-container {
+      height: 80px;
+    }
+
+    .current-user-avatar {
+      border-radius: 100%;
+      height: inherit;
+      object-fit: cover;
+      border: 2px solid var(--base-90);
+    }
+
+    h3 {
+      margin-top: 6px;
```

Comment:
This is my mistake in the spec (some version controlling issues). Instead of using `6px`, can we use `$su-2`? That's apart of Crayons' spacing unit variables, which you can find in `stylesheets/config/_import.scss`.

Reply:
Updated!